### PR TITLE
BZ2014790: [Docs] [DisasterRecovery] highlight that failovers cannot occur if the SPM is the point of failure if Power Management is not configured

### PR DESCRIPTION
**Pull request by stoobie**
Bug fix

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=2014790

Changes proposed in this pull request:
- Updated topics on Storage consideration as per https://bugzilla.redhat.com/show_bug.cgi?id=2014790#c1:
  - https://github.com/oVirt/ovirt-site/pull/2653/files#diff-f07c76203bba9d6afcb62d5b09264276dd2e011fc6b39b98057025ccf5d4415e 
  - https://github.com/oVirt/ovirt-site/pull/2653/files#diff-840c2951c1ffae1691ee0fa5bf02f1e75fb24f0aeefde0abeeafc7e1dd8c3e86
- Lots of other editing fixes, not connected to technical content.

Preview
https://ovirt.github.io/ovirt-site/previews/2653/documentation/disaster_recovery_guide/index.html#storage_considerations_active_active

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @stoobie 
This pull request needs review by: @sleviim

### CODE REVIEW

File: source/documentation/disaster_recovery_guide/task/task_ConfigureStretchCluster.adoc [hunk 1]
```diff
@@ -42,7 +42,7 @@ The standalone {engine-name} is only highly available when managed externally. F
 * In a public cloud.
 ====
 
-*Configuring the Standalone {engine-name} Stretch Cluster*
+.Procedure
```
Comment (barpavel) [inline source/documentation/disaster_recovery_guide/task/task_ConfigureStretchCluster.adoc:45]: IMHO too generic word, unless that's the common practice in RH documentation.

File: source/documentation/disaster_recovery_guide/task/task_CreateAnsiblePlaybooks.adoc [hunk 2]
```diff
@@ -13,9 +13,12 @@ Ansible is used to initiate and manage the disaster recovery failover and failba
 ** Networks with the same general connectivity as the primary site.
 * Replicated storage. See <<storage_considerations_active-passive>> for more information.
 +
-NOTE: The replicated storage that contains virtual machines and templates must not be attached to the secondary site.
+[NOTE]
```
Comment (barpavel) [inline source/documentation/disaster_recovery_guide/task/task_CreateAnsiblePlaybooks.adoc:16]: Again, old style had an icon and a "NOTE" word.
The new style is better / more common?

File: source/documentation/disaster_recovery_guide/task/task_CreateAnsiblePlaybooks.adoc [hunk 3]
```diff
@@ -13,9 +13,12 @@ Ansible is used to initiate and manage the disaster recovery failover and failba
...
+====
+The replicated storage that contains virtual machines and templates must not be attached to the secondary site.
 * The `oVirt.disaster-recovery` package must be installed on the highly available Red Hat Ansible Engine machine that will automate the failover and failback.
```
Comment (barpavel) [inline source/documentation/disaster_recovery_guide/task/task_CreateAnsiblePlaybooks.adoc:19]: You made this and next sentences as part of the above comment...
Previously they were separate bullets.

File: source/documentation/disaster_recovery_guide/task/task_CreateCleanupPlaybook.adoc [hunk 4]
```diff
@@ -1,12 +1,12 @@
 [[create_cleanup]]
-==== Create the Playbook to Clean the Primary Site
+= Create the Playbook to Clean the Primary Site
 
-Before you failback to the primary site, you need to ensure that the primary site is cleaned of all storage domains to be imported. This can be performed manually on the {engine-name}, or optionally you can create an Ansible playbook to perform it for you.
+Before you failback to the primary site, you must ensure that the primary site is cleaned of all storage domains to be imported. You can do this manually on the {engine-name}, or optionally you can create an Ansible playbook to do it for you.
```
Comment (barpavel) [inline source/documentation/disaster_recovery_guide/task/task_CreateCleanupPlaybook.adoc:4]: Although I'm not a native English speaker, but IMHO "to do it for you" should be:
"to perform it for you"
or
"to execute it for you"

File: source/documentation/disaster_recovery_guide/task/task_ExecuteFailback.adoc [hunk 5]
```diff
@@ -13,7 +13,7 @@ Once you fail over, you can fail back to the primary site when it is active and
 
 This example uses the `dr-rhv-failback.yml` playbook created earlier.
 
-*Executing a failback*:
+.Procedure
```
Comment (barpavel) [inline source/documentation/disaster_recovery_guide/task/task_ExecuteFailback.adoc:16]: Again, IMHO the old was better.

File: source/documentation/disaster_recovery_guide/task/task_ExecuteFailover.adoc [hunk 6]
```diff
@@ -10,17 +10,20 @@
...
+[IMPORTANT]
+====
+Sanlock must release all storage locks from the replicated storage domains before the failover process starts.
 These locks should be released automatically approximately 80 seconds after the disaster occurs.
+====
 
 This example uses the `dr-rhv-failover.yml` playbook created earlier.
 
-*Executing a failover*:
+.Procedure
```
Comment (barpavel) [inline source/documentation/disaster_recovery_guide/task/task_ExecuteFailover.adoc:21]: Same comment as above.

File: source/documentation/disaster_recovery_guide/task/task_ExecuteFailover.adoc [hunk 7]
```diff
@@ -10,17 +10,20 @@
...
 
-Run the failover playbook with the following command:
+. Run the failover playbook with the following command:
```
Comment (barpavel) [inline source/documentation/disaster_recovery_guide/task/task_ExecuteFailover.adoc:23]: This and next sentences have "1." numbering both.

File: .gitignore [hunk 8]
```diff
@@ -36,3 +36,4 @@ source/.jekyll-metadata
 
 # ignore docs index.html: index.adoc is used instead
 source/documentation/*/index.html
+source/documentation/*/build/
```
[the remark this answers is not in the document]
Comment (stoobie) [reply .gitignore:39]: Fixed.

File: source/documentation/administration_guide/topics/Manually_fencing_or_isolating_a_nonresponsive_host.adoc [hunk 9]
```diff
@@ -5,11 +5,11 @@ If a host unpredictably goes into a non-responsive state, for example, due to a
 
 [WARNING]
 ====
-Do not use the *Confirm host has been rebooted* option unless you have manually rebooted the host. Using this option while the host is still running can lead to a virtual machine image corruption.
+Do not select *Confirm 'Host has been Rebooted'* unless you have manually rebooted the host. Using this option while the host is still running can lead to a virtual machine image corruption.
 ====
 
 
-*Manually fencing or isolating a non-responsive host*
```
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/administration_guide/topics/Manually_fencing_or_isolating_a_nonresponsive_host.adoc:12]: I made several edits, including this one, to align with the (Modular Documentation Reference Guide)[https://redhat-documentation.github.io/modular-docs/]. This heading is redundant, it just repeats the name of the topic. There are lots of similar occurrences.

File: source/documentation/disaster_recovery_guide/assembly/asm_ActivePassiveDR.adoc [hunk 10]
```diff
@@ -1,14 +1,12 @@
 [[active_passive]]
-== Active-Passive Disaster Recovery
+= Active-Passive Disaster Recovery
 :cptdir: ../concept
 :taskdir: ../task
 
-This chapter provides instructions to configure {virt-product-fullname} for disaster recovery using the active-passive disaster recovery solution.
```
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/assembly/asm_ActivePassiveDR.adoc:6]: Yes, it's self-referential and it doesn't add anything useful. The title of the section provides the same information.

File: source/documentation/disaster_recovery_guide/concept/cpt_DisasterRecoverySolutions.adoc [hunk 11]
```diff
@@ -1,14 +1,14 @@
 [[disaster_recovery_solutions]]
 
-== Disaster Recovery Solutions
+= Disaster Recovery Solutions
 
-{virt-product-fullname} (RHV) supports two types of disaster recovery solutions to ensure that environments can recover when a site outage occurs. Both solutions support two sites, and both require replicated storage.
+{virt-product-fullname} supports two types of disaster recovery solutions to ensure that environments can recover when a site outage occurs. Both solutions support two sites, and both require replicated storage.
 
-*Active-Active Disaster Recovery*:
+.Active-Active Disaster Recovery
```
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/concept/cpt_DisasterRecoverySolutions.adoc:7]: I understand what you mean, and I don't disagree with you. But I'm not focused right now on the look of a given output. Just to give you some insight, it used to be years ago that a tech writer would spend a lot of time thinking about the look and feel in addition to the content. Nowadays, in a scalable solution such as ours, writers focus on content, and designers focus on the look and feel. So I'm concerned with the proper way to structure this content. The "dot" notation I used here is semantically correct for minor headings such as this one. The bold notation that you prefer the look of is simply bold notation, but it's not semantically correct.

The right way to deal with what you're talking about is to address it in the CSS. However, that's outside the scope of this bug. If you feel it's really important, could you open a new issue about it? That way it'll get some notice and might be addressed. I don't know who owns the CSS for the site.

File: source/documentation/disaster_recovery_guide/task/task_CreateAnsiblePlaybooks.adoc (hunk 3 above)
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/task/task_CreateAnsiblePlaybooks.adoc:19]: Good catch! Fixed.

File: source/documentation/disaster_recovery_guide/task/task_CreateCleanupPlaybook.adoc (hunk 4 above)
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/task/task_CreateCleanupPlaybook.adoc:4]: I actually was trying to make it simper language, and also to keep it parallel with the "do" earlier in the sentence:

"You can *do* this manually" ... "to *do* it for you."

Using "perform/execute" in one, but not both of these cases would, at least for me, make the sentence less clear (using the same word to refer to the same thing is a rule in technical writing). And for me, using "do" is just as clear as "perform" or "execute" and is smoother and less stiff. When I try "perform" or "execute", it almost gets in my way. This is very subjective, of course.

File: source/documentation/disaster_recovery_guide/task/task_ExecuteFailback.adoc (hunk 5 above)
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/task/task_ExecuteFailback.adoc:16]: Same response as before.

File: source/documentation/disaster_recovery_guide/task/task_ExecuteFailover.adoc (hunk 6 above)
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/task/task_ExecuteFailover.adoc:21]: Same response.

File: source/documentation/disaster_recovery_guide/task/task_ExecuteFailover.adoc (hunk 7 above)
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/task/task_ExecuteFailover.adoc:23]: Good catch! Fixed.

File: source/documentation/disaster_recovery_guide/concept/cpt_DisasterRecoverySolutions.adoc [hunk 12]
```diff
@@ -1,14 +1,14 @@
...
 
 
-*Active-Passive Disaster Recovery*:
+.Active-Passive Disaster Recovery
 
-Also referred to as site-to-site failover, this disaster recovery solution is implemented by configuring two seperate RHV environments: the active primary environment, and the passive secondary (backup) environment. Failover and failback between sites must be manually executed, and is managed by Ansible. See <<active_passive_overview>> for more information.
+Also referred to as site-to-site failover, this disaster recovery solution is implemented by configuring two separate RHV environments: the active primary environment, and the passive secondary (backup) environment. Failover and failback between sites must be manually executed, and is managed by Ansible. See xref:active_passive_overview[Active-Passive Overview] for more information.
```
[the remark this answers is not in the document]
Comment (RichardHoch) [reply source/documentation/disaster_recovery_guide/concept/cpt_DisasterRecoverySolutions.adoc:14]: @stoobie I noticed the RHV also. Please either leave it alone or chnage it and then mark this conversation as resolved,

**Comment by RichardHoch** [comment]
@stoobie -- a number of minor comments, but otherwise, LGTM!

File: source/documentation/disaster_recovery_guide/concept/cpt_DisasterRecoverySolutions.adoc (hunk 12 above)
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/concept/cpt_DisasterRecoverySolutions.adoc:14]: Fixed.

File: source/documentation/disaster_recovery_guide/concept/cpt_NetworkConsiderationsAA.adoc [hunk 13]
```diff
@@ -1,6 +1,6 @@
 [[network_considerations]]
-==== Network Considerations
+= Network Considerations
```
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/concept/cpt_NetworkConsiderationsAA.adoc:2]: I addressed the module hierarchy in the parent assembly. If you look at the preview, you should see that these sections are still sub-sections of 2.1.

File: source/documentation/disaster_recovery_guide/concept/cpt_StorageConsiderationsAA.adoc [hunk 14]
```diff
@@ -1,8 +1,20 @@
 [[storage_considerations_active_active]]
-==== Storage Considerations
+= Storage Considerations
```
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/concept/cpt_StorageConsiderationsAA.adoc:2]: They still should be sub-sections in the preview.

File: source/documentation/disaster_recovery_guide/concept/cpt_StorageConsiderationsAA.adoc [hunk 15]
```diff
@@ -1,8 +1,20 @@
...
+The storage domain for {virt-product-fullname} can comprise either block devices (SAN - iSCSI or FCP) or a file system (NAS - NFS, GlusterFS, or other POSIX compliant file systems). For more information about {virt-product-fullname} storage see link:{URL_virt_product_docs}{URL_format}administration_guide/index#chap-Storage[Storage] in the _Administration Guide_.
 
 The sites require synchronously replicated storage that is writeable on both sites with shared layer 2 (L2) network connectivity. The replicated storage is required to allow virtual machines to migrate between sites and continue running on the site’s storage. All storage replication options supported by {enterprise-linux} 7 and later can be used in the stretch cluster.
 
-IMPORTANT: If you have a custom multipath configuration that is recommended by the storage vendor, see the instructions and important limitations in link:{URL_virt_product_docs}{URL_format}installing_{URL_product_virt}_as_a_self-hosted_engine_using_the_command_line/index#proc-Customizing_Multipath_Configurations_for_SAN_Vendors_SHE_cli_deploy[Customizing Multipath Configurations for SAN Vendors].
+[IMPORTANT]
```
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/concept/cpt_StorageConsiderationsAA.adoc:8]: You should still see that icon and IMPORTANT text. This is just a different AsciiDoc format that supports multiple lines. The other format only supports a single line. I was just trying to bring some more consistency into the source code.

File: source/documentation/disaster_recovery_guide/concept/cpt_StorageConsiderationsAA.adoc [hunk 16]
```diff
@@ -1,8 +1,20 @@
...
+====
+
+Set the SPM role on a host at the primary site to have precedence. To do so, configure SPM priority as high in the primary site hosts and SPM priority as low on secondary site hosts. If you have a primary site failure that impacts network devices inside the primary site, preventing the fencing device for the SPM host from being reachable, such as power loss, the hosts in the seconday site are not able to take over the SPM role.
+
+In such a scenario virtual machines do a failover, but operations that require the SPM role in place cannot be executed, including adding new disks, extending existing disks, and exporting virtual machines.
+
+To restore full functionality, detect the actual nature of the disaster and after fixing the root cause and rebooting the SPM host, select *Confirm 'Host has been Rebooted'* for the SPM host.
```
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/concept/cpt_StorageConsiderationsAA.adoc:17]: @barpavel 
What I understand from Gianlucca's comment is:
If the SPM host isn't working and the secondary site cannot take over the SPM role, then you need to get the SPM host back online. In order to do that, you need to fix whatever the problem is.

If you understand differently, then please tell me what you think should be here.

File: source/documentation/disaster_recovery_guide/concept/cpt_StorageConsiderationsAP.adoc [hunk 17]
```diff
@@ -1,12 +1,15 @@
...
-IMPORTANT: Local storage domains are unsupported for disaster recovery.
+[IMPORTANT]
+====
+Local storage domains are unsupported for disaster recovery.
+====
 
 A primary and secondary storage replica is required. The primary storage domain’s block devices or shares that contain virtual machine disks or templates must be replicated. The secondary storage must not be attached to any data center, and will be added to the backup site’s data center during failover.
 
-If you are implementing disaster recovery using self-hosted engine, ensure that the  storage domain used by the self-hosted engine {engine-name} virtual machine does not contain virtual machine disks because the storage domain will not be failed over.
+If you are implementing disaster recovery using a self-hosted engine, ensure that the  storage domain used by the {engine-name} virtual machine does not contain virtual machine disks because in such a case the storage domain will not be failed over.
```
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/concept/cpt_StorageConsiderationsAP.adoc:13]: Yes, I see it. It shouldn't matter as far as output is concerned, but I'll fix it.

File: source/documentation/disaster_recovery_guide/concept/cpt_NetworkConsiderationsAA.adoc [hunk 18]
```diff
@@ -1,6 +1,6 @@
...
 
-All hosts in the cluster must be on the same broadcast domain over an L2 network. This means that connectivity between the two sites needs to be L2.
+All hosts in the cluster must be on the same broadcast domain over an L2 network. So connectivity between the two sites must be L2.
```
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/concept/cpt_NetworkConsiderationsAA.adoc:4]: I actually prefer "So" over "Therefore" almost always. It's simpler and flows off my tongue much better, is less formal, although still proper, and means the same thing.

File: source/documentation/disaster_recovery_guide/concept/cpt_StorageConsiderationsAA.adoc [hunk 19]
```diff
@@ -1,8 +1,20 @@
...
+[IMPORTANT]
+====
+If you have a custom multipath configuration that is recommended by the storage vendor, see the instructions and important limitations in link:{URL_virt_product_docs}{URL_format}installing_{URL_product_virt}_as_a_self-hosted_engine_using_the_command_line/index#proc-Customizing_Multipath_Configurations_for_SAN_Vendors_SHE_cli_deploy[Customizing Multipath Configurations for SAN Vendors].
+====
+
+Set the SPM role on a host at the primary site to have precedence. To do so, configure SPM priority as high in the primary site hosts and SPM priority as low on secondary site hosts. If you have a primary site failure that impacts network devices inside the primary site, preventing the fencing device for the SPM host from being reachable, such as power loss, the hosts in the seconday site are not able to take over the SPM role.
```
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/concept/cpt_StorageConsiderationsAA.adoc:13]: The guidance is to use bold for GUI text. This is not GUI text. Also, guidance is to _not_ use commands/UI text as meaningful words within a sentence. So if these _are_ words that appear in the interface, then I would have to write e.g.:

To do so, select *High* from the *SPM priority* popup menu...

This particular module is not really a procedure, and it's not really telling the user how to set the priority to high. Honestly, that's kind of a bug in this module (imho). But I'm not dealing with that here and now.

File: source/documentation/disaster_recovery_guide/concept/cpt_StorageConsiderationsAA.adoc [hunk 20]
```diff
@@ -1,8 +1,20 @@
...
+
+In such a scenario virtual machines do a failover, but operations that require the SPM role in place cannot be executed, including adding new disks, extending existing disks, and exporting virtual machines.
```
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/concept/cpt_StorageConsiderationsAA.adoc:15]: Eh...could go either way. This sentence already has so many commas, I'll leave it as-is.

File: source/documentation/disaster_recovery_guide/concept/cpt_StorageConsiderationsAP.adoc [hunk 21]
```diff
@@ -1,12 +1,15 @@
 [[storage_considerations_active-passive]]
-==== Storage Considerations
+= Storage Considerations
 
-The storage domain for {virt-product-fullname} can be made of either block devices (SAN - iSCSI or FCP) or a file system (NAS - NFS, GlusterFS, or other POSIX compliant file systems). For more information about {virt-product-fullname} storage see link:{URL_virt_product_docs}{URL_format}administration_guide/index#chap-Storage[Storage] in the _Administration Guide_. 
+The storage domain for {virt-product-fullname} can comprise either block devices (SAN - iSCSI or FCP) or a file system (NAS - NFS, GlusterFS, or other POSIX compliant file systems). For more information about {virt-product-fullname} storage see link:{URL_virt_product_docs}{URL_format}administration_guide/index#chap-Storage[Storage] in the _Administration Guide_.
```
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/concept/cpt_StorageConsiderationsAP.adoc:4]: You're right. It's got this same casing in the title of the module.

File: source/documentation/disaster_recovery_guide/reference/rfr_discreet_failover.adoc [hunk 22]
```diff
@@ -1,7 +1,7 @@
 [[discreet_failover]]
-=== Discreet Failover Test
+= Discreet Failover Test
 
-This test simulates a failover while the primary site and all its storage domains remain active, allowing users to continue working in the primary site. For this to happen you will need to disable replication between the primary storage domains and the replicated (secondary) storage domains. During this test the primary site will be unaware of the failover activities on the secondary site.
+This test simulates a failover while the primary site and all its storage domains remain active, allowing users to continue working in the primary site. For this to happen you must disable replication between the primary storage domains and the replicated (secondary) storage domains. During this test the primary site will be unaware of the failover activities on the secondary site.
```
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/reference/rfr_discreet_failover.adoc:4]: 1. At the same time.
2. Good suggestion. I made a change.

File: source/documentation/disaster_recovery_guide/reference/rfr_non_discreet_failover_failback.adoc [hunk 23]
```diff
@@ -1,11 +1,11 @@
 [[non_discreet_failover_failback]]
-=== Full Failover and Failback test
+= Full Failover and Failback test
 
 This test performs a full failover and failback between the primary and secondary site. You can simulate the disaster by shutting down the primary site's hosts or by adding firewall rules to block writing to the storage domains.
 
 For more information about failing over the environment, cleaning the environment, and performing the failback, see <<execute_failover>>, <<clean>>, and <<execute_failback>>.
 
-.Procedure: Failover test:
+.Procedure: Failover test
```
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/reference/rfr_non_discreet_failover_failback.adoc:8]: Yeah, I know what you mean. The thing is that these two sections are both procedures without any intro text. So it doesn't really make sense to add sub-headings and then immediately begin the procedures, either with or without the .Procedure.
I could invest more time and make the sub-heading and come up with some intro text between the sub-heading and the .Procedure...but I don't think anyone gains anything from that.

File: source/documentation/disaster_recovery_guide/task/task_Clean.adoc [hunk 24]
```diff
@@ -1,22 +1,22 @@
...
+* Synchronize the replication from the secondary site's storage domains to the primary site's storage domains.
 * Clean the primary site of all storage domains to be imported. This can be done manually in the {engine-name}, or by creating and running an Ansible playbook. See link:{URL_virt_product_docs}{URL_format}administration_guide/index#Detaching_a_storage_domain[Detaching a Storage Domain] in the _Administration Guide_ for manual instructions, or <<create_cleanup>> for information to create the Ansible playbook.
 
 This example uses the `dr-cleanup.yml` playbook created earlier to clean the environment.
 
-*Cleaning the primary site:*
+.Procedure
 
-Clean up the primary site with the following command:
+. Clean up the primary site with the following command:
```
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/task/task_Clean.adoc:15]: Yes, I thought I fixed this, but I see that I didn't. Take a look now.

File: source/documentation/disaster_recovery_guide/task/task_CommonStepsConfigureStretchCluster.adoc [hunk 25]
```diff
@@ -1,6 +1,6 @@
 . Configure the SPM priority to be higher on all hosts in the primary site to ensure SPM failover to the secondary site occurs only when all hosts in the primary site are unavailable. See link:{URL_virt_product_docs}{URL_format}administration_guide/index#SPM_Priority[SPM Priority] in the _Administration Guide_.
 
-. Configure all virtual machines that need to failover as highly available, and ensure that the virtual machine has a lease on the target storage domain. See link:{URL_virt_product_docs}{URL_format}virtual_machine_management_guide/index#Configuring_a_highly_available_virtual_machine[Configuring a Highly Available Virtual Machine] in the _Virtual Machine Management Guide_.
+. Configure all virtual machines that must failover as highly available, and ensure that the virtual machine has a lease on the target storage domain. See link:{URL_virt_product_docs}{URL_format}virtual_machine_management_guide/index#Configuring_a_highly_available_virtual_machine[Configuring a Highly Available Virtual Machine] in the _Virtual Machine Management Guide_.
```
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/task/task_CommonStepsConfigureStretchCluster.adoc:3]: :)

File: source/documentation/disaster_recovery_guide/task/task_ConfigureStretchCluster.adoc (hunk 1 above)
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/task/task_ConfigureStretchCluster.adoc:45]: It's standard practice at Red Hat. It's prescribed in the Modular Documentation Reference Guide.

File: source/documentation/disaster_recovery_guide/task/task_CreateAnsiblePlaybooks.adoc [hunk 26]
```diff
@@ -1,7 +1,7 @@
 [[create_ansible_playbooks]]
-=== Create the Required Ansible Playbooks
+= Create the Required Ansible Playbooks
 
-Ansible is used to initiate and manage the disaster recovery failover and failback. You therefore need to create Ansible playbooks to facilitate this. For more information about creating Ansible playbooks, see the link:http://docs.ansible.com/ansible/latest/playbooks.html[Ansible documentation].
+Ansible is used to initiate and manage the disaster recovery failover and failback. You therefore must create Ansible playbooks to facilitate this. For more information about creating Ansible playbooks, see the link:http://docs.ansible.com/ansible/latest/playbooks.html[Ansible documentation].
```
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/task/task_CreateAnsiblePlaybooks.adoc:4]: Changed to "So you must"

File: source/documentation/disaster_recovery_guide/task/task_CreateFailoverFailbackPlaybooks.adoc [hunk 27]
```diff
@@ -1,5 +1,5 @@
 [[create_failover_failback]]
-==== Create the Failover and Failback Playbooks
+= Create the Failover and Failback Playbooks
```
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/task/task_CreateFailoverFailbackPlaybooks.adoc:2]: This isn't really a task. It's an example. I think it requires more thought...which I'm not doing right now.

File: source/documentation/disaster_recovery_guide/task/task_CreateFailoverFailbackPlaybooks.adoc [hunk 28]
```diff
@@ -22,7 +22,7 @@ dr_sites_primary_password: primary_password
 dr_sites_secondary_password: secondary_password
 ----
 
-NOTE: For extra security you can encrypt the password file. However, you will need to use the `--ask-vault-pass` parameter when running the playbook. See _Using Ansible Roles to Configure {virt-product-fullname}_ in the link:{URL_virt_product_docs}{URL_format}administration_guide/index#Using_Ansible_Roles[Administration Guide] for more information.
+NOTE: For extra security you can encrypt the password file. However, you must use the `--ask-vault-pass` parameter when running the playbook. See _Using Ansible Roles to Configure {virt-product-fullname}_ in the link:{URL_virt_product_docs}{URL_format}administration_guide/index#Using_Ansible_Roles[Administration Guide] for more information.
```
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/task/task_CreateFailoverFailbackPlaybooks.adoc:25]: This note formatting is actually valid for notes that are a single line. Take a look at the preview or the published content. That said, I do prefer to use the other notation that you're suggesting and I've made several such changes. So why not here too?

File: source/documentation/disaster_recovery_guide/task/task_ExecuteFailover.adoc [hunk 29]
```diff
@@ -10,17 +10,21 @@
 ** The `oVirt.disaster-recovery` package.
 ** The mapping file and required failover playbook.
 
-IMPORTANT: Sanlock must release all storage locks from the replicated storage domains before the failover process starts.
+[IMPORTANT]
+====
+Sanlock must release all storage locks from the replicated storage domains before the failover process starts.
```
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/task/task_ExecuteFailover.adoc:15]: I don't think it's a keyword, I think it's a mechanism.

File: source/documentation/disaster_recovery_guide/task/task_GenerateMapping.adoc [hunk 30]
```diff
@@ -29,7 +31,10 @@ In this example the Ansible playbook is named `dr-rhv-setup.yml`, and is execute
     - oVirt.disaster-recovery
 ----
 +
-NOTE: For extra security you can encrypt your {engine-name} password in a `.yml` file. See link:{URL_virt_product_docs}{URL_format}administration_guide/index[Using Ansible Roles to Configure {virt-product-fullname}] in the _Administration Guide_ for more information.
+[NOTE]
+====
+For extra security you can encrypt your {engine-name} password in a `.yml` file. See link:{URL_virt_product_docs}{URL_format}administration_guide/index[Using Ansible Roles to Configure {virt-product-fullname}] in the _Administration Guide_ for more information.
```
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/task/task_GenerateMapping.adoc]: Done.

File: source/documentation/disaster_recovery_guide/task/task_CreateCleanupPlaybook.adoc [hunk 31]
```diff
@@ -1,12 +1,12 @@
 [[create_cleanup]]
-==== Create the Playbook to Clean the Primary Site
+= Create the Playbook to Clean the Primary Site
 
-Before you failback to the primary site, you need to ensure that the primary site is cleaned of all storage domains to be imported. This can be performed manually on the {engine-name}, or optionally you can create an Ansible playbook to perform it for you.
+Before you failback to the primary site, you must ensure that the primary site is cleaned of all storage domains to be imported. You can do so manually on the {engine-name}, or optionally you can create an Ansible playbook to do it for you.
```
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/task/task_CreateCleanupPlaybook.adoc]: Fixed.

File: source/documentation/disaster_recovery_guide/task/task_CreateCleanupPlaybook.adoc [hunk 32]
```diff
@@ -1,12 +1,12 @@
 [[create_cleanup]]
-==== Create the Playbook to Clean the Primary Site
+= Create the Playbook to Clean the Primary Site
```
[the remark this answers is not in the document]
Comment (stoobie) [reply source/documentation/disaster_recovery_guide/task/task_CreateCleanupPlaybook.adoc:2]: I'm leaving as is. This isn't really a tast, it's a code example.

**Review by RichardHoch — APPROVED** [review]

**Pull request merged**